app/scripts/modules/core/src/presentation/forms/CurrentForm.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { Field, FieldProps, FormikProps } from 'formik';
+
+interface ICurrentFormProps<T> {
+  render: (props: FormikProps<T>) => React.ReactNode;
+}
+
+/**
+ * This component provides access to the current Formik `form` props.
+ * This can be useful to access current form data outside of a FormField or Input.
+ *
+ * Example:
+ * ```js
+ * <CurrentForm render={formik => {
+ *   const username = formik.values.username;
+ *   return (
+ *     <h1>Hello {username}</h1>
+ *   )
+ * }} />
+ * ```
+ */
+export class CurrentForm<T = any> extends React.Component<ICurrentFormProps<T>> {
+  public render() {
+    return <Field name="" render={(props: FieldProps<T>) => this.props.render(props.form)} />;
```

Comment:
what does `name=""` do?

Reply:
I wanted a way to get access to the current `form` and used `<Field>` to do so because it's public API.  I think `name` is required prop to `<Field>`.  I'll check the docs and remove if it's not required.

Reply:
Yes, it's required prop for `Field`.  It's basically a hack to get the `form` from a `Field` but not specify any actual field name.